src/components/FollowersFollowingUsers.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import UserAvatar from "./UserAvatar";
+import { Button } from "antd";
+
+interface FollowersFollowingUsersProps {
+  selectedListTitle: string;
+  tempUserList: any[];
+}
+
+const FollowersFollowingUsers = ({
+  selectedListTitle,
+  tempUserList,
+}: FollowersFollowingUsersProps): JSX.Element => {
+  const userList = () => {
+    switch (selectedListTitle) {
+      case "showFollowers":
+        return tempUserList.filter((user) => user.followsMe);
+      case "showFollowing":
+        return tempUserList.filter((user) => user.following);
+      default:
+        return [];
```

Comment:
Another reason to use the enum is that you don't have to have a default case! (It would instead be used with ListStatus.CLOSED and TS will check for completeness in the switch statement)